_data/sites.json

```
@@ -1158,6 +1158,16 @@
         ]
     },
 
+    {
+        "name": "AniDB",
+        "url": "https://anidb.net/perl-bin/animedb.pl?show=policies",
+        "difficulty": "hard",
+        "notes": "Go to the page linked and send an email to the address found under \"Privacy - Removal of Your Account\".",
```

```suggestion
        "notes": "Needs to contact customer support.",
        "email": "info@anidb.net",
```
We can put an email on the entry, because then users will see a "send e-mail" button :)